source/extensions/filters/listener/http_inspector/http_inspector.cc

```diff
@@ -147,6 +149,7 @@ ParseState Filter::parseHttpHeader(absl::string_view data) {
         // Set other HTTP protocols to HTTP/1.0
         protocol_ = Http::Headers::get().ProtocolStrings.Http10String;
       }
+      ENVOY_LOG(trace, "http inspector: set application protocol to %s", protocol_);
```

Comment:
I think we want this in `done()`. `protocol_` and `protocol` are not the same (primarily different casing)

Reply:
I will update it.

Reply:
So this one can be removed, right?

Reply:
So this one can be removed, right?